### PR TITLE
fixes ISSUE-130, ISSUE-182

### PR DESCRIPTION
**Pull request by javamonkey79**
* fix to allow user pagination
* fix to allow logging by config

### CODE REVIEW

**Comment by javamonkey79** [comment]
This also fixes ISSUE-194, ISSUE-195

File: slacker.py [hunk 1]
```diff
@@ -241,22 +255,17 @@ def post_message(self, channel, message, message_type=None):
...
-        bot_avatar_url = self.config.bot_avatar_url
-        if bot_name or bot_avatar_url:
-            post_data['as_user'] = False
-            if bot_name:
-                post_data['username'] = bot_name
-            if bot_avatar_url:
-                post_data['icon_url'] = bot_avatar_url
-
         if message_type:
-            post_data['attachments'] = json.dumps([{'fallback': message_type}], encoding='utf-8')
```
Comment (kmarekspartz) [inline slacker.py:259]: Is this now the default?

Comment (javamonkey79) [reply slacker.py:259]: yes, I think it is a side effect of running on python 3

**Comment by h3h** [comment]
I'm guessing this is largely defunct. Please re-open if the intent is to true it up.

**Pull request closed without merging**